EventListener/AngularCsrfCookieListener.php

```diff
@@ -102,7 +102,9 @@ public function onKernelResponse(FilterResponseEvent $event)
             $this->cookiePath,
             $this->cookieDomain,
             $this->cookieSecure,
-            false
+            false /* httpOnly */,
+            false /* raw */,
+            null  /* sameSite */
```

Comment:
Shouldn't we set this flag to `lax` at least?

Reply:
Yes, I agree, security first. I set it to `null` initially as this was the default behavior in previous versions. This has been done.

Reply:
@dunglas This value is now set to `lax` instead of `null`.